lesson_7_hw/task_3.1.js

```diff
@@ -0,0 +1,16 @@
+// 5*. Напишите функцию findMissingNumber(arr), которая принимает массив чисел от 1 до N (исключая одно число)
+//   и возвращает пропущенное число. Массив не отсортирован и может содержать дубликаты.
+//   Решите эту задачу, используя эффективные методы массива.
+function findMissingNumber(arr) {
```

Comment:
If interesting, try to refactor for handling the case where two or more numbers is absent